--- origsrc/geocode-glib-3.10.0/geocode-glib/geocode-glib.symbols	2013-09-17 09:24:56.000000000 -0500
+++ src/geocode-glib-3.10.0/geocode-glib/geocode-glib.symbols	2014-03-24 13:06:03.482347900 -0500
@@ -23,9 +23,7 @@ geocode_reverse_resolve
 geocode_error_quark
 geocode_error_get_type
 _geocode_parse_search_json
-_geocode_parse_resolve_json
 _geocode_read_nominatim_attributes
-_geocode_ip_json_to_location
 _geocode_create_place_from_attributes
 _geocode_place_set_icon
 geocode_place_get_type
